crates/turbo-tasks-memory/src/stats.rs

```diff
@@ -99,7 +99,11 @@ impl Stats {
     }
 
     pub fn add(&mut self, backend: &MemoryBackend, task: &Task) {
-        self.add_conditional(backend, task, |_, info| info.executions > 0)
+        self.add_conditional(backend, task, |_, info| {
+            info.executions
+                .map(|executions| executions > 0)
+                .unwrap_or(true)
```

Comment:
It's fine. It's only 0 when you reset the stats or during execution. Otherwise every task has executed.
You can also see from `last_duration` if a task has executed.